CMake/ITKSetStandardCompilerFlags.cmake

```diff
@@ -304,16 +352,59 @@ macro(check_compiler_platform_flags)
   endif()
 endmacro()#End the platform check function
 
+if(NOT ITK_C_WARNING_FLAGS OR NOT ITK_CXX_WARNING_FLAGS ) # Only check once if not explicitly set on command line
+  #-----------------------------------------------------------------------------
+  #Check the set of warning flags the compiler supports
+  check_compiler_warning_flags(C_WARNING_FLAGS CXX_WARNING_FLAGS)
+endif()
+if(NOT ITK_C_WARNING_FLAGS) #Not set on cmake command line option -DITK_C_WARNING_FLAGS:STRING=""
+  set(ITK_C_WARNING_FLAGS ${C_WARNING_FLAGS} CACHE STRING "ITK C Compiler Warning flags, Modify to suit your needs.")
```

Comment:
Either put `.` instead of `,`, or modify should be lowercase. Also copy-pasted a few time below.

Reply:
cleaned, and reorganized to remove duplications.